extensions/blocks/podcast-player/style.scss

```diff
@@ -1,11 +1,148 @@
 /**
- * Frontend (site) styles for Podcast Episodes
+ * Podcast Player block shared styles (editor & front-end).
  */
 
+$episode-v-padding: 15px;
+$episode-h-padding: 10px;
+$episode-status-icon-size: 22px;
+$text-color: #92959a;
+$text-color-hover: #000;
+$text-color-active: $text-color-hover;
+$text-color-error: #db5051;
+$block-bg-color: #fff;
```

Comment:
Are those colors we could pull from existing WordPress or Automattic colors instead of setting them here?
https://github.com/Automattic/jetpack/blob/c9e2901cd559ec75322abd21a50d6e00eddbee78/extensions/README.md#L229-L231

Reply:
Yes! Done. Thank you.